### DISCUSSION TITLE
Schedule not running between 1am and 2am?

### DISCUSSION THREAD

**Discussion by agonzalezm** [quartznet/quartznet#665] [General] [answered]
I have this code and i am scheduling a job to run every 15 minutes every day but i see that it get executed every 15 minutes except beween 1am and 2am. Any idea why scheduler doesnt run everyday between 1am and 2am?

var days = GetDaysOfWeek(jobScheduler.DaysOfWeek).ToArray();
                triggerBulider.WithDailyTimeIntervalSchedule(x => x
                              .WithMisfireHandlingInstructionIgnoreMisfires()
                              .OnDaysOfTheWeek(days)
                              .StartingDailyAt(new TimeOfDay(jobScheduler.StartDateTime.Hour, jobScheduler.StartDateTime.Minute))
                              .EndingDailyAt(new TimeOfDay(jobScheduler.EndDateTime.Hour, jobScheduler.EndDateTime.Minute))
                              .WithIntervalInMinutes(jobScheduler.IntervalType == IntervalType.Minutes ? jobScheduler.IntervalValue : jobScheduler.IntervalValue * 60));

**Comment by lahma** [comment, marked as answer]
In such cases should use `TriggerUtils.ComputeFireTimes` and check how trigger behaves.
